integrations/huggingface-transformers/tutorials/question-answering/question_answering-cli.md

```diff
@@ -0,0 +1,372 @@
+observer_epoch: 12.0
+quantize_embeddings: 1
+
+distill_hardness: 1.0
+distill_temperature: 5.0
+
+weight_decay: 0.0
+
+# Modifiers:
+training_modifiers:
+  - !EpochRangeModifier
+      end_epoch: eval(num_epochs)
+      start_epoch: 0.0
+
+  - !LearningRateFunctionModifier
+      start_epoch: 0
+      end_epoch: eval(num_epochs)
+      lr_func: linear
+      init_lr: eval(init_lr)
+      final_lr: eval(final_lr)
+
+quantization_modifiers:
+  - !QuantizationModifier
+      start_epoch: eval(qat_start_epoch)
+      disable_quantization_observer_epoch: eval(observer_epoch)
+      freeze_bn_stats_epoch: eval(observer_epoch)
+      quantize_embeddings: eval(quantize_embeddings)
+      quantize_linear_activations: 0
+      exclude_module_types: ['LayerNorm']
+      submodules:
+        - bert.embeddings
+        - bert.encoder
+        - qa_outputs
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [start_logits, end_logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+
+regularization_modifiers:
+  - !SetWeightDecayModifier
+      start_epoch: 0.0
+      weight_decay: eval(weight_decay)
+```
+
+</details>
+
+You can download the recipe with the following code:
+
+```python
+from sparsezoo import Model
+transfer_stub = "zoo:nlp/question_answering/obert-base/pytorch/huggingface/squad/pruned90_quant-none"
+download_dir = "./transfer_recipe-squad"
+zoo_model = Model(transfer_stub, download_path=download_dir)
+recipe_path = zoo_model.recipes.default.path
+print(recipe_path)
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process 
+are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path obert_base_pruned90_quant_squad \
+  --task question_answering
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning with a Custom Dataset (SquadShifts Amazon)
+
+Beyond the SQuAD task, we can also pass a dataset from the Hugging Face Hub or pass via local files. Let's try an example with  for the extractive question answering using [Squadshifts Amazon Dataset](https://huggingface.co/datasets/squadshifts), which containing ~10,000 question answer pairs from the Amazon product reviews.
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation
+modifiers, by setting `--distill_teacher disable` we instruct SparseML to skip distillation.
+
+### Squadshifts Dataset Inspection
+
+Run the following to inspect the Rotten Tomatoes dataset.
```

Comment:
Shouldn't this and the code below reference [Squadshifts Amazon Dataset](https://huggingface.co/datasets/squadshifts) instead of the rotten tomatoes dataset?